test_utils/utils/setup_fixtures.py

```diff
@@ -172,3 +178,49 @@ def create_item_groups(settings, only_create=None):
 		ig = frappe.new_doc("Item Group")
 		ig.update(item_group)
 		ig.save()
+
+
+def create_workstations(only_create=None):
+	workstations = get_fixtures_data_from_file(filename="workstations.json")
+
+	for workstation in workstations:
+		if only_create and workstation.get("workstation_name") not in only_create:
+			continue
+
+		if frappe.db.exists("Workstation", workstation.get("workstation_name")):
+			continue
+
+		workstation_doc = frappe.new_doc("Workstation")
+		workstation_doc.update(workstation)
+		workstation_doc.save()
+
+
+def create_operations(only_create=None):
+	operations = get_fixtures_data_from_file(filename="operations.json")
+
+	for operation in operations:
+		if only_create and operation.get("name") not in only_create:
+			continue
+
+		if frappe.db.exists("Operation", operation.get("name")):
+			continue
+
+		operation_doc = frappe.new_doc("Operation")
+		operation_doc.update(operation)
+		operation_doc.save()
+
+
+def create_boms(settings, only_create=None):
+	boms = get_fixtures_data_from_file(filename="boms.json")
+
+	for bom in boms:
+
+		if only_create and bom.get("item") not in only_create:
+			continue
+
+		if frappe.db.exists("BOM", {"item": bom.get("item")}):
```

Comment:
We a two BOMs in the dataset for the same item on purpose. We can omit this check.